R/packages.R

```diff
@@ -11,21 +11,7 @@ resolve_dependencies <- function(pkgs, local = TRUE) {
   unique(inst$get_resolution()$package)
 }
 
-get_default_wasm_assets <- function(desc) {
-  pkg <- desc$Package
-  r_wasm <- "http://repo.r-wasm.org"
-  # TODO: Restore the use of short version, once webR with R 4.4.0 is released.
-  #       This function can then be merged with `get_r_universe_wasm_assets()`
-  r_short <- WEBR_R_VERSION
-  contrib <- glue::glue("{r_wasm}/bin/emscripten/contrib/{r_short}")
-
-  info <- utils::available.packages(contriburl = contrib)
-  if (!pkg %in% rownames(info)) {
-    cli::cli_warn("Can't find {.pkg {pkg}} in webR binary repository.")
-    return(list())
-  }
-  ver <- info[pkg, "Version", drop = TRUE]
-
+warn_repo_pkg_version <- function(desc, ver) {
```

Comment:
Small nit: this function's name could reflect it doesn't _always_ warn. 

```suggestion
check_repo_pkg_version <- function(desc, ver) {
```

Or something like this

```suggestion
warn_if_repo_pkg_version_mismatch <- function(desc, ver) {
```

Reply:
Thanks! I like `check_repo_pkg_version()`.